src/sanka/Translator.java

```diff
@@ -32,13 +36,17 @@ public static void main(String[] argv) throws Exception {
         }
         translator.parse(contextList);
         if (env.errorCount > 0) {
-            return;
+            System.exit(CANNOT_PARSE);
         }
         translator.evaluate();
         if (env.errorCount > 0) {
-            return;
+            System.exit(CANNOT_EVALUATE);
+        }
+        try {
+            translator.translate();
+        } catch (IOException ioe) {
+            System.exit(IO_EXCEPTION_TRANSLATING);
```

Comment:
What have you got against the exit status of an uncaught Exception? If you let it go uncaught you probably get a stack trace which is nice.